docs/tutorials/projects/integrating-viam-with-openai.md

```diff
@@ -243,13 +227,13 @@ Make sure `Path to Existing Model on Robot` is selected.
 Update the **Model Path** and **Label Path** to match where you [copied the tutorial software](#5-set-up-tutorial-software).
 For example, the model path would would be similar to:
 
-```bash
+``` bash
 /home/<username>/tutorial-openai-integration/detector/effdet0.tflite
 ```
 
 and the label path similar to:
 
-```bash
+``` bash
```

Comment:
```suggestion
```sh {class="command-line" data-prompt="$"}
```

Reply:
done, just not applying inline to fix other things